.golangci.yml

```diff
@@ -79,6 +80,13 @@ linters-settings:
   enable:
     - shadow
 
+  depguard:
+    rules:
+      main:
+        deny:
+          - pkg: "^math/rand$"
+            desc: "Use math/rand/v2 instead"
+
```

Comment:
👍 Good idea. I'm thinking we should review our deps and see whether there are other packages we may want to keep consistent. Thinking of YAML parsers and logging.